x-pack/plugins/security_solution/server/lib/detection_engine/signals/search_after_bulk_create.ts

```diff
@@ -96,17 +98,27 @@ export const searchAfterAndBulkCreate = async ({
   // signalsCreatedCount keeps track of how many signals we have created,
   // to ensure we don't exceed maxSignals
   let signalsCreatedCount = 0;
+  let totalToFromTuples;
+
+  try {
+    totalToFromTuples = getSignalTimeTuples({
+      logger,
+      ruleParamsFrom: ruleParams.from,
+      ruleParamsTo: ruleParams.to,
+      ruleParamsMaxSignals: ruleParams.maxSignals,
+      gap,
+      previousStartedAt,
+      interval,
+      buildRuleMessage,
+    });
```

Comment:
`getSignalTimeTuples` will also throw an error now if the "from" on the rule param is NaN, hence why we are now wrapping it in a catch and ejecting from this function early with a failure state in the `catch`